src/config/2019.json

```diff
@@ -695,6 +695,14 @@
       "github": "KJLarson",
       "twitter": "KaJLa47"
     },
+    "tymosh": {
+      "name": "Kate Tymoshkina",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "https://avatars2.githubusercontent.com/u/25080897?v=4&s=200",
+      "github": "tymosh"
+    },
```

Comment:
You happy with this @tymosh ? You can also add Website, Twitter and LinkedIn profiles if you want.

Here's [what it looks like on the site](https://20210116t081702-dot-webalmanac.uk.r.appspot.com/uk/2019/contributors#tymosh):

![tymosh profile](https://user-images.githubusercontent.com/10931297/104807051-a0a7ce80-57d3-11eb-85c6-dba9377e2919.png)

Reply:
Please, add LinkedIn account: https://www.linkedin.com/in/tymosh/

Reply:
```suggestion
    "tymosh": {
      "name": "Kate Tymoshkina",
      "teams": [
        "translators"
      ],
      "avatar_url": "https://avatars2.githubusercontent.com/u/25080897?v=4&s=200",
      "github": "tymosh",
      "linkedin": "tymosh"
    },
```